pyEpiabm/pyEpiabm/toy_population_config.py

```diff
@@ -0,0 +1,94 @@
+import numpy as np
+from .population import Population
+from .household import Household
+
+
+class ToyPopulation:
```

Comment:
Can we rename this to ToyPopulationFactory or ToyPopulationMaker or ToyPopulationConstructor?
Class itself isn't a ToyPopulation, but instead is a helper to construct a simple Population.